create-new-data-pr.py

```diff
@@ -35,29 +45,39 @@ def update_tag_version(current_version=None):
   data_prid = int(opts.pull_request)
   dist_repo = gh.get_repo(opts.dist_repo)
   data_repo_pr = data_repo.get_pull(data_prid)
-  data_repo_base_branch = data_repo_pr.base.ref
+  data_pr_base_branch = data_repo_pr.base.ref
+  data_repo_default_branch = data_repo.default_branch
   # create master just to exist on the cms-data repo if it doesn't
-  if data_repo_base_branch == "main":
+  if data_repo_default_branch == "main":
       if "master" not in [branch.name for branch in data_repo.get_branches()]:
-          data_repo.create_git_ref(ref='refs/heads/master', sha=data_repo.get_branch(data_repo_base_branch).commit.sha)
+          data_repo.create_git_ref(ref='refs/heads/master', sha=data_repo.get_branch(data_repo_default_branch).commit.sha)
 
-  if (data_repo_base_branch != "master") and (data_repo_base_branch != "main"):
-      print("I do not know how to tag a non-master (non-main) branch %s" % data_repo_pr.base.ref)
-      exit(1)
+  last_release_tag = None
+  repo_name_only = opts.data_repo.split("/")[1]
+  #find the last tag on the default branch, which is needed in all cases
```

Comment:
@mrodozov , I do not understand this comment? why tag on the pr branch is not enough?

Reply:
in case we just branched out the default branch as another barnch, made PR on top of it and merged it the last tags of the non-default and the default branch will be the same, and need to be compared. 
Example is - we branch out 10_6_X from master, but now 10_6_X is the same as master. If we make a PR from 10_6_X and merge it, new tag will be created with greater number than the one existing on master, which shouldn't happen. So comparison with the default branch is always needed. Also if the branch is the default - the other branch doesn't go trough checks so it's not even redundant

Reply:
We do not branch off from the tip of default. We keep on adding/merging on default branch. We only create a new branch when there are already changes in default which should not go in the non-default branch. So in practice we only create a new branch from an old tag.